src/params.rs

```diff
@@ -61,6 +62,26 @@ pub struct RunParameters {
                             // HOME: /
 }
 
+impl RunParameters {
+    /// Examines the local network interfaces, and tries to find our assigned IP within the data network.
```

Comment:
```suggestion
    /// Examines the local network interfaces, and tries to find our assigned IP within the data network.
    ///
```

Otherwise I don't think this would be rendered as a new paragraph.